pom.xml

```diff
@@ -132,5 +132,10 @@
           <artifactId>apache-httpcomponents-client-4-api</artifactId>
           <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>trilead-api</artifactId>
```

Comment:
??

Reply:
needed for testing

Reply:
Testing…what?

Reply:
for using JenkinsRule (or similar)

Reply:
Right, but why this plugin, which does not appear to have anything to do with the changes you are making? Smells like a POM mistake.